### PR TITLE
Added Release Notes

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ X] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-05-11-traces.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+keywords:
+  - traces
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We've enhanced the Trace Query Visualizations screen by adding five new aggregation charts alongside the existing Trace duration breakdown chart. These charts provide more insights into the behavior and performance of your application. Aggregating traces allows you to quickly identify anomalies and unexpected behaviors, resulting in a shorter Time-to-Resolution (TTR) and a higher Return on Investment (ROI).
+
```
Comment (pbrzoska-sumo) [inline blog-service/2023-05-11-traces.md:13]:
```suggestion
The new aggregate charts are available as time-series or histogram and cover following 3 metrics:
* Trace duration
* Error count
* Span count

```

File: blog-service/2023-05-11-traces.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+The new aggregate charts are:
+* Trace duration
+* Error count
+* Span count
+* Trace duration as timeseries
+* Trace duration as histogram
+
+To access the new charts, simply navigate to the Trace Query Visualizations screen and select the **Trace duration** dropdown menu. From there, you can choose from the **duration/errors/spans** (per trace) and **timeseries/histogram** options. With these choices, you'll now have a total of six charts to help you better understand the profile of your traces.
```
Comment (pbrzoska-sumo) [inline blog-service/2023-05-11-traces.md:21]:
```suggestion
To access the new charts, simply navigate to the Traces screen and after selecting a subset of traces that are comparable (e.g. same transaction type) click on "Show Chart". Using dropdown menus you can choose from the **duration/errors/spans** (per trace) and **timeseries/histogram** options. With these choices, you'll now have a total of six charts to help you better understand the profile of your traces.
```

File: blog-service/2023-05-11-traces.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+NOTE: We recommend filtering your trace data to represent transactions for the best results.
```
Comment (pbrzoska-sumo) [inline blog-service/2023-05-11-traces.md:23]:
```suggestion
```

File: blog-service/2023-05-11-traces.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+[Learn More](https://help.sumologic.com/docs/apm/traces/view-and-investigate-traces/#trace-duration-breakdown-chart)
```
Comment (kimsauce) [inline blog-service/2023-05-11-traces.md:25]: While you were out on leave, we switched from absolute to rel. links in release notes.

```suggestion
[Learn More](/docs/apm/traces/view-and-investigate-traces/#trace-duration-breakdown-chart)
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
one fix

File: blog-service/2023-05-11-traces.md (hunk 4 above)
Comment (JainM6) [reply blog-service/2023-05-11-traces.md:33]: @kimsauce Thank you. That makes sense. Style should be the same for all the docs.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**